main.rb

```diff
@@ -4,21 +4,19 @@
 
```

Comment:
- [ ] OPTIONAL:  The SOLID can be improved, I'll explain:

## Explanation

Every class should have a single responsibility and methods that relate to the main functionality of that class.  Also, the user input, the "puts", and the interface should be apart from the logic.  That way, if you add more functionality in the future, you won't need to refactor the whole thing, thus making the app more easily scalable.

## Hints

- All requests for input and input validation should be placed in the `main.rb` file and/or classes/methods whose responsibilities are to request or validate input.  That way, we comply with the single responsibility rule.

- Also, the `app.rb` files and modules, should just serve as "listers" to populate the screen with data that the user requests (like show all books, etc).